data/hai/midas.txt

```diff
@@ -0,0 +1,1010 @@
+			choice
+				`	(This is it. I want to know all.)`
+				`	(Ignorance is bliss.)`
+					goto bliss
+			`	They take a deep breath. "An important organization wants these gold bars you have on board. They're not just normal gold. If you were to sell them to me, I could give you a very generous price of 1000000 credits per bar, and you wouldn't be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have every single one. I may have told you too much for them to allow me to live already."`
+			choice
+				`	(Sell the bars.)`
+				`	(Keep the bars, and drop that traitor on the planet.)`
+					goto bliss2
+				`	(Keep the crew member for further interrogation.)`
+					goto escape
+				`	(Make them walk the plank.)`
+					to display
+						has "outfit (flagship installed): Heshi"
+					goto heshi
+			action
+				outfit "Gold Bars" -5
+				payment 5000000
+				set "midas: sold bars"
+			`	The soon-to-be gone traitor nods. "It is the right choice you are making, Captain <last>. You will never be bothered by any of this again, that I promise. I will take these 5 piles, and any more will be sold little by little as you go on most worlds. If you sell them to any normal outfitters, we will easily get our hands on it, so it is no problem to us, but you would only be paid the raw value of gold for it."`
+				decline
+			label bliss2
+			`	You leave the traitor here on this world after awakening them, and they seem thankful but deeply troubled by what happened. You wonder what you should do with the gold bars that seem to cause so much trouble. They should probably be stashed away safely, or maybe you should get rid of them.`
+				decline
+			label heshi
+			action
+				"midas: proven worthy" ++
+			`	You put the traitor in the aquarium and the Heshi takes care of them. This should send a strong message to Midas you are not one to be trifled with.`
+				decline
+			label escape
+			`	You tighten your hold on the traitor, but as you do something hits the back of your head, and you lose consciousness. When you wake, there is a small note on the spot where some gold bars were, written by yet another member of your crew. "<first>, I am very sorry to break your trust like they did, but believe me, this is for your own good. The ex-crew member would have been killed if they had returned without the gold bars, so we took a few. You should quit this while you still can."`
+			label lostbars
+			action
+				outfit "Gold Bars" -5
+	# accept is used when you spot the crew member but do not know the link to the bars. If they are revealed, they will not try again.
+	on accept
+		fail "Midas: Betrayal For The Bars"
+
+# Sell the bars if you agreed to the deal, 1 million a piece and 5 by 5 followed by one by one so we can get all of them.
+mission "Midas: Bar Selling"
```

Comment:
Doesn’t seem like this is quite fair if first last wants a few souvenirs (or if they want an easy source of disabled ships or combat rating)

Reply:
it only happens after you accept doing it

Reply:
That’s true. This can be resolved.